estimator/transport_co2/model.py

```diff
@@ -60,6 +58,21 @@ def __init__(self, avg_co2_per_vehicle_km: float, avg_occupancy: float):
     SCOOTER = (86.4, 1.2)
     BUS = (863, 12.7)
 
+    # Additional OTP modes
+    WALK = (0, 1)
+    BICYCLE = (0, 1)
+    CAR = (mean([SMALL_CAR[0], LARGE_CAR[0]]), 1.5)
```

Comment:
It seems like importing `mean` and creating a `list` wrapper around the two values is a little overkill for simply dividing by 2. It's fine if it increases readability for you, but I think it makes it a little harder.

Reply:
Using 'mean' makes it explicit what we are trying to achieve. Dividing by two leaves it to the reader to deduce why we are dividing by two, and would encourage a code comment to explain the 'why'. Adding a bit if overhead from standard lib is worth it IMO to improve readability and make the code self-documting. One of the main reasons why we use Python is because of this ethos of Developer friendliness expressed in the Zen of Python - rather than prioritizing performance as would be preferred in languages like Go or Rust.